doc/modules/calibration.rst

```diff
@@ -84,6 +84,16 @@ typical for maximum-margin methods (compare Niculescu-Mizil and Caruana [1]_),
 which focus on difficult to classify samples that are close to the decision
 boundary (the support vectors).
 
+Confidence intervals for the fraction of positives in each bin can be drawn
```

Comment:
I don't understand how it's possible to derive a valid confidence interval from the standard deviation.

If you want to use the standard error of the mean (stddev / sqrt(n)), it's only possible for normally distributed random variables. Here the we have a binomial distribution (for the binary classification outcome in each bin).

https://en.wikipedia.org/wiki/Standard_error#Assumptions_and_usage

Reply:
We can specify that this is a normal approximation (by applying the CLT), which isn't a poor one for sample proportions

https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Normal_approximation_interval

Reply:
Interesting refs:

- https://stats.stackexchange.com/questions/4756/confidence-interval-for-bernoulli-sampling
- https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval

Reply:
Another interesting reference suggesting that proportion CIs aren't so simple after all:

* https://math.stackexchange.com/questions/1448233/the-derivation-of-the-wald-interval

I'll make some changes and hopefully find some good references. Thanks for your comments!